### PR TITLE
Release Firestore Emulator v1.14.1.

### LINKED ISSUE #2421: firebase exports cannot be imported on other platforms
Creating an export of windows and importing it on macOS gives an IO error.

### [REQUIRED] Environment info

**firebase-tools:** 8.4.3

**Platform:** macOS and Windows

### [REQUIRED] Steps to reproduce

1. Create an export on windows and then import it in mac.

### [REQUIRED] Expected behavior

1. Should be able to run with all the imports.

### [REQUIRED] Actual behavior

I get an IO error:

```
Exception in thread "main" com.google.cloud.datastore.core.exception.DatastoreException: /Users/akshayrajgollahalli/Documents/GitHub/accounting-ui/mock-data/firestore_export/all_namespaces\all_kinds\all_namespaces_all_kinds.export_metadata (No such file or directory)
	at com.google.cloud.datastore.emulator.impl.ExportImportUtil.parseBackupFile(ExportImportUtil.java:312)
	at com.google.cloud.datastore.emulator.impl.ExportImportUtil.fetchEntities(ExportImportUtil.java:60)
	at com.google.cloud.datastore.emulator.firestore.CloudFirestore.main(CloudFirestore.java:89)
Caused by: java.io.FileNotFoundException: /Users/akshayrajgollahalli/Documents/GitHub/accounting-ui/mock-data/firestore_export/all_namespaces\all_kinds\all_namespaces_all_kinds.export_metadata (No such file or directory)
	at java.base/java.io.FileInputStream.open0(Native Method)
	at java.base/java.io.FileInputStream.open(FileInputStream.java:212)
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:154)
	at com.google.cloud.datastore.emulator.impl.ExportImportUtil.parseBackupFile(ExportImportUtil.java:308)
	... 2 more
```

### PR DESCRIPTION
**Pull request by xwkuang5**
### Description

This PR updates Firestore Emulator to v1.14.1.

### Scenarios Tested

Started emulators and it downloads and verifies the checksum correctly.

```
node ../firebase-tools/lib/bin/firebase.js emulators:start
i  emulators: Starting emulators: auth, functions, firestore, database
⚠  functions: The following emulators are not running, calls to these services from the Functions emulator will affect production: hos
ting, pubsub, storage
✔  functions: Using node@12 from host.
i  firestore: downloading cloud-firestore-emulator-v1.14.1.jar...
Progress: ============================================================================================================> (100% of 61MB)
i  firestore: Removing outdated emulator files: cloud-firestore-emulator-v1.13.1.jar
i  firestore: Firestore Emulator logging to firestore-debug.log
⚠  database: Did not find a Realtime Database rules file specified in a firebase.json config file. The emulator will default to allowi
ng all reads and writes. Learn more about this option: https://firebase.google.com/docs/emulator-suite/install_and_configure#security_
rules_configuration.
i  database: Database Emulator logging to database-debug.log
i  ui: downloading ui-v1.6.5.zip...
Progress: =============================================================================================================> (100% of 4MB)
i  ui: Removing outdated emulator files: ui-v1.6.4
i  ui: Removing outdated emulator files: ui-v1.6.4.zip
i  ui: Emulator UI logging to ui-debug.log
```

Fixes #2421

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+*   Release Cloud Firestore emulator v1.14.0
+    *   Chore: prepare for aggregate support.
```
Comment (yuchenshi) [inline CHANGELOG.md:2]: Is this a user-facing change? If not, please remove.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+    *   Chore: add support of x-goog-request-params http header for routing.
+    *   Fix: change read-past-max-staleness error code to align with production
+        implementation.
+    *   Fix: update readtime-in-the-future error message.
+    *   Fix: support importing exports from Windows on UNIX systems.
```
Comment (yuchenshi) [inline CHANGELOG.md:7]:
```suggestion
    *   Fix: support importing exports from Windows on UNIX systems. (#2421)
```

**Review by yuchenshi — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (xwkuang5) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+- Release Cloud Firestore emulator v1.14.0
+  - Chore: add support of x-goog-request-params http header for routing.
+  - Fix: change read-past-max-staleness error code to align with production
+    implementation.
+  - Fix: update readtime-in-the-future error message.
+  - Fix: support importing exports from Windows on UNIX systems. (#2421)
```
Comment (bkendall) [inline CHANGELOG.md:6]:
```suggestion
- Release Cloud Firestore emulator v1.14.0:
  - Adds support of x-goog-request-params http header for routing.
  - Changes `read-past-max-staleness` error code to align with production
    implementation.
  - Updates readtime-in-the-future error message.
  - Supports importing exports from Windows on UNIX systems. (#2421)
```

**Pull request closed without merging**